### ISSUE TITLE
Yuzu crashes immediately in the overworld - Legend of Zelda TOTK

### ISSUE THREAD

**Issue by Baggos1234** [yuzu-emu/yuzu#10905]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Affected Build(s)

Mainline 1478 / EA 3708

### Description of Issue

The game instantly crashes when I try to load a save in the overworld. 
Loading a game in small areas (starting area or inside shrines) works and then immediately crashes when going out in the overworld
It affects vulkan only. OpenGL works, albeit with graphical glitches and very low performance. ATSC DECODER is disabled
I've tried this with versions 1.0, 1.1 and 1.2 of the game, with and without mods (DFPS, blackscreen fix, visual improvements) and with different settings configuration (probably tried every combination). It also affects the previous yuzu builds, I just used the latest here

This might be a similar issue https://github.com/yuzu-emu/yuzu/issues/10239 but the log files shows different errors (I don't get a devise loss occurred error). Also that issue is closed.

I have low VRAM (2 gb only) but if I'm not mistaken ASTC recompression with BC1 should make that ok

It could be a driver issue since my gpu is old and has legacy drivers (Driver Version
21.09.13.02-220802a-382934C-RadeonSoftware). If this is the problem kindly point it out so I can rest assured at least

### Expected Behavior

Not crashing

### Reproduction Steps

1. Load a save in a small area
2. Go outside in the overworld
3. Crash

[slot_00.zip](https://github.com/yuzu-emu/yuzu/files/11859584/slot_00.zip)

### Log File

In this log file it ends with the error Debug <Critical> core\hle\service\filesystem\fsp_srv.cpp:operator ():899: Assertion Failed! [yuzu_log.txt](https://github.com/yuzu-emu/yuzu/files/11859651/yuzu_log.txt)

In previous log files it ended halfway through Render.Vulkan <Info> video_core\renderer_vulkan\vk_pipeline_cache.cpp:CreateGraphicsPipeline:586: 

### System Configuration

CPU: Ryzen 5 1400
GPU/Driver: AMD R7 360 / Driver Version
21.09.13.02-220802a-382934C-RadeonSoftware
RAM: 16 GB 
OS: Windows 10

**Comment by Squall-Leonhart** [comment]
> Available VRAM: 1.47 GiB


This is just not going to happen for you.

**Comment by DeadKiller25** [comment]
> > Available VRAM: 1.47 GiB
> 
> This is just not going to happen for you.

Same thing is happening to me, I have 4GB VRAM, AMG GPU user, still too low?

**Comment by ReaperOQ** [comment]
Same thing. AMD APU(R5 5600H), 16GB RAM

**Comment by Squall-Leonhart** [comment]
windows itself uses video memory.

**Comment by Baggos1234** [comment]
> > Available VRAM: 1.47 GiB
> 
> This is just not going to happen for you.

That bugged me as well.. Do you have any idea why my available vram is lower than the actual one? I have literally no other programs open 

> windows itself uses video memory.

True but wouldn't that be accounted for when the developers talk about a 2 Gb vram GPU? I mean.. The BC1 recompression - according to developer's posts - should allow people to run games on a 2Gb GPU (not 2 Gb available Vram)

**Comment by jedinjapan** [comment]
You are pushing the very lower limits of what could possibly run the game at this point. It seems quite clearly to point to a VRAM constraint.

If you booted your system into Linux and ran yuzu from a light distro, you might be able to manage on 2GB VRAM, but with Window 10+ you are going to be over the limit in certain areas unless you find a mod to lower texture quality or model detail potentially.

**Comment by DeadKiller25** [comment]
> 

What about twice the VRAM amount? I have the same error on log and 4GB, is it due to VRAM?

**Comment by Baggos1234** [comment]
> > 
> 
> What about twice the VRAM amount? I have the same error on log and 4GB, is it due to VRAM?

It shouldn't be due to low vram, if you have bc1 recompression enabled. Do you also have an old gpu with legacy drivers? Cause that may also be the issue

**Comment by Baggos1234** [comment]
> You are pushing the very lower limits of what could possibly run the game at this point. It seems quite clearly to point to a VRAM constraint.
> 
> If you booted your system into Linux and ran yuzu from a light distro, you might be able to manage on 2GB VRAM, but with Window 10+ you are going to be over the limit in certain areas unless you find a mod to lower texture quality or model detail potentially.

Hm sounds right.
I have also tried with Linux (kubuntu) but I can't even manage to boot the game into the smaller areas, so that's a whole other issue

**Comment by DeadKiller25** [comment]
> > > 
> > 
> > 
> > What about twice the VRAM amount? I have the same error on log and 4GB, is it due to VRAM?
> 
> It shouldn't be due to low vram, if you have bc1 recompression enabled. Do you also have an old gpu with legacy drivers? Cause that may also be the issue

Yes, it's an R7 450, altough I replaced the legacy drivers with the Amernime zone ones, didn't work for TOKT but it fixed other PC games which previously had graphical errors (Doom Eternal e.g)

**Comment by Baggos1234** [comment]
> > > > 
> > > 
> > > 
> > > What about twice the VRAM amount? I have the same error on log and 4GB, is it due to VRAM?
> > 
> > 
> > It shouldn't be due to low vram, if you have bc1 recompression enabled. Do you also have an old gpu with legacy drivers? Cause that may also be the issue
> 
> Yes, it's an R7 450, altough I replaced the legacy drivers with the Amernime zone ones, didn't work for TOKT but it fixed other PC games which previously had graphical errors (Doom Eternal e.g)

Huh.. I didn't know custom drivers existed. I may try those if I can't find anything else wrong. Though I doubt that will solve the problem, considering Linux open source drivers don't either.

**Comment by DeadKiller25** [comment]
> 

I may have installed these the wrong way, there's supposed to be a visual cue on the system tray icon but it's the same as always. Also, I did try Linux but I was unable to activate amdgpu drivers :(

**Comment by nessie17a** [comment]
I have the same problem here, AMD R7 450 4GB RAM.
Crashes after loading screen in overworld. 

Before that it ran fine till the Ganondorf Cutscene.
